ui/components/MesheryPatterns.js

```diff
@@ -779,6 +782,18 @@ function PatternForm({ pattern, onSubmit, show }) {
     setYaml(data.valueOf().getValue())
   }
 
+  function insertPattern(workload) {
+    const attrName = getPatternAttributeName(workload);
+    var returnValue = {}
+    Object.keys(deployServiceConfig).forEach(key => {
+      if (deployServiceConfig[key]['type'] === attrName) {
+        returnValue = deployServiceConfig[key]
```

Comment:
arent you trying to find here? can we use the `Object.keys(object).find` instead?

Reply:
Yeah, sounds good.

Reply:
Done :smile: